3rdparty/python/requirements.txt

```diff
@@ -4,7 +4,7 @@ freezegun==1.1.0
 
 # Note: we use humbug to report telemetry. When upgrading, ensure the new version maintains the
 # anonymity promise we make here: https://www.pantsbuild.org/docs/anonymous-telemetry
-humbug==0.1.9
+humbug==0.2.6
```

Comment:
The CI errors are because you have to edit constraints.txt as well as requirements.txt.

Reply:
Thanks @benjyw . Fixed.